admin/workspace-providers/deploying-workspace-provider.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Deployment
+description: Learn how to deploy a workspace provider.
+state: beta
+---
+
+This article walks you through the process of deploying a workspace provider to
+a [Kubernetes cluster](../../setup/kubernetes/index.md).
+
+[Workspace Providers](index.md) are logical groups of resources to which
+developers can deploy workspaces. Like the Coder deployment, workspace providers
```

Comment:
Everywhere in this document that it says "workspaces" should be "environments", minus "Workspace Providers".

Reply:
Agreed